src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -71,7 +71,7 @@ private void CollectPInvokes(List<PInvoke> pinvokes, List<PInvokeCallback> callb
             }
             catch (Exception ex) when (ex is not LogAsErrorException)
             {
-                Log.LogWarning(null, "WASM0001", "", "", 0, 0, 0, 0, $"Could not get pinvoke, or callbacks for method '{method.Name}' because '{ex.Message}'");
+                Log.LogWarning(null, "WASM0001", "", "", 0, 0, 0, 0, $"Could not get pinvoke, or callbacks for method '{type.FullName}::{method.Name}' because '{ex.Message}'");
```

Comment:
A similar message is also here https://github.com/dotnet/runtime/blob/main/src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs#L304 and in the icall generator https://github.com/dotnet/runtime/blob/main/src/tasks/WasmAppBuilder/IcallTableGenerator.cs#L155

Reply:
Fixed